afew/MailMover.py

```diff
@@ -19,7 +19,7 @@ class MailMover(Database):
     '''
 
 
-    def __init__(self, max_age=0, rename = False, dry_run=False):
+    def __init__(self, max_age=0, rename = False, dry_run=False, no_new=False):
```

Comment:
I'd prefer to call this variable `skip_notmuch_new`. It's a bit more explaining.